src/ngraph/serializer.cpp

```diff
@@ -427,7 +427,7 @@ static shared_ptr<ngraph::Function>
                 else
                 {
                     node = make_shared<op::BatchNorm>(
-                        epsilon, args[0], args[1], args[2], args[3], args[4]);
```

Comment:
We'll need to update the serializer to handle the five-arg training variant as well.

Reply:
(On a re-read, my comment was not very clear.) What I mean is that this won't correctly deserialize the five-argument version of the op when `training` is `true`.